test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/OtlpLogExporterTests.cs

```diff
@@ -589,6 +589,34 @@ public void CheckToOtlpLogRecordBodyIsPopulated(bool includeFormattedMessage)
         Assert.Equal("state", otlpLogRecord.Body.StringValue);
     }
 
+#if EXPOSE_EXPERIMENTAL_FEATURES && NET8_0_OR_GREATER
```

Comment:
I don't think `EXPOSE_EXPERIMENTAL_FEATURES` works in test projects. Typically they use `internal`s when things aren't exposed. But it looks like we don't do `InternalsVisibleTo` from in-memory exporter. Here's a way you can make it work which shouldn't need any preprocessor directives or extra `InternalsVisibleTo` definitions:

```csharp
    [Fact]
    public void LogRecordBodyIsExportedWhenUsingBridgeApi()
    {
        var logRecords = new List<LogRecord>();

        using (var loggerProvider = Sdk.CreateLoggerProviderBuilder()
            .ConfigureServices(services =>
            {
                services.AddLogging(builder => builder
                    .AddOpenTelemetry(options => options.AddInMemoryExporter(logRecords)));
            })
            .Build())
        {
            var logger = loggerProvider.GetLogger();

            logger.EmitLog(new LogRecordData()
            {
                Body = "Hello world",
            });
        }

        Assert.Single(logRecords);

        var otlpLogRecordTransformer = new OtlpLogRecordTransformer(DefaultSdkLimitOptions, new());

        var otlpLogRecord = otlpLogRecordTransformer.ToOtlpLog(logRecords[0]);

        Assert.Equal("Hello world", otlpLogRecord.Body?.StringValue);
    }
```